cedar-policy/src/ffi/validate.rs

```diff
@@ -155,28 +147,16 @@ impl ValidationCall {
 }
 
 /// Configuration for the validation call
-#[derive(Serialize, Deserialize, Debug)]
+#[derive(Serialize, Deserialize, Debug, Default)]
 #[cfg_attr(feature = "wasm", derive(tsify::Tsify))]
 #[cfg_attr(feature = "wasm", tsify(into_wasm_abi, from_wasm_abi))]
 #[serde(rename_all = "camelCase")]
 #[serde(deny_unknown_fields)]
 pub struct ValidationSettings {
-    /// Whether validation is enabled. If this flag is set to `false`, then
-    /// only parsing is performed. The default value is `true`.
-    enabled: bool,
     /// Used to control how a policy is validated. See comments on [`ValidationMode`].
     mode: ValidationMode,
 }
 
-impl Default for ValidationSettings {
-    fn default() -> Self {
-        Self {
-            enabled: true,
-            mode: ValidationMode::default(),
-        }
-    }
-}
```

Comment:
Might make sense to keep this around?

Reply:
It's still there - it's just been moved to an automatic derivation of `Default`.